src/Microsoft.AspNetCore.SignalR.Common/Protocol/IHubProtocol.cs

```diff
@@ -57,4 +57,9 @@ public interface IHubProtocol
         /// <returns>A value indicating whether the protocol supports the specified version.</returns>
         bool IsVersionSupported(int version);
     }
+
+    public interface IHubProtocolWithMinorVersion : IHubProtocol
```

Comment:
`IHubProtocol2`? Adding a number is how interfaces are recommended to be versioned. Prevents in the future:

`IHubProtocolWithMinorVersionAndSomethingElseWeJustAddedFiveYearsFromNow`

Reply:
This is new territory for us. We don't have any `InterfaceName2` interfaces around. But I do generally agree.